x-pack/platform/plugins/shared/cases/server/cases_analytics/analytics_index.test.ts

```diff
@@ -109,6 +111,11 @@ describe('AnalyticsIndex', () => {
         ...mappings,
         _meta: mappingsMeta,
       },
+      aliases: {
+        [indexAlias]: {
+          is_write_index: true,
```

Comment:
My understanding is that because the alias points only to a single index, this will be defaulted to true anyway.
@ymao1 Christos said you might be able to help with this. Do you have some insight?